hazelcast/src/main/java/com/hazelcast/internal/partition/impl/MigrationManager.java

```diff
@@ -1090,7 +1120,6 @@ private void migrationOperationFailed(Member partitionOwner) {
             } finally {
                 partitionServiceLock.unlock();
             }
-            partitionService.getPartitionEventManager().sendMigrationEvent(migrationInfo, MigrationEvent.MigrationStatus.FAILED);
```

Comment:
Is this now sent in `finally` block in `MigrateTask#run`? Same for `migrationOperationSucceeded`.

Reply:
Yes, correct.